x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/closure_options_radio.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useCallback } from 'react';
+import { EuiRadioGroup } from '@elastic/eui';
+
+import * as i18n from './translations';
+
+const idPrefix = 'closure_options';
```

Comment:
nit: I think for that type of variables we used to use uppercase letters with an underscore, so I think it should be `const ID_PREFIX = 'closure_options';`